emcache/_cython/protocol/ascii/multi_line_parser.pyx

```diff
@@ -101,3 +103,6 @@ cdef class AsciiMultiLineParser:
 
     def cas(self):
         return self.cas_
+
+    def value(self):
```

Comment:
can we just store the client error into a new class string [here](https://github.com/emcache/emcache/pull/146/files#diff-4e9b092a9ed62d31cb339036c6c0a3c5f4fb6fed1005cd5d564139c77347bb77R40)

Like:

```
self.client_error_ = .....
self.future.set_result()
```

Then [here](https://github.com/emcache/emcache/blob/2cc04a7db4aec5f4e23b39be814cfe94ec44dce6/emcache/protocol.py#L163) 

```
keys, values, flags, cas, client_error = parser.keys(), parser.values(), parser.flags(), parser.cas(), parser.client_error()
```

All values will be `[]` except the `client_error`